model/account.go

```diff
@@ -38,11 +38,14 @@ func (a *InternalAccount) NameKey() *datastore.Key {
 
 func (a *InternalAccount) Load(ps []datastore.Property) error {
 	tmp := struct {
-		ID        string
-		ProjectID string
-		Address   []byte
-		DraftCode string
-		Index     int
+		ID                string
+		ProjectID         string
+		Address           []byte
+		DraftCode         string
+		Index             int
+		DeployedCode      any // leave it for backward compatibility
```

Comment:
old db models had this, so we need it in order for datastore to load successfully